test/unit/queue/src/QueueTest.cpp

```diff
@@ -26,6 +26,28 @@ using TestQueues = alpaka::meta::Concatenate<
 #endif
     >;
 
+//! Equivalent too CHECK but conditions with false negatives
```

Comment:
Typo: should be `to` instead of `too`. I would also rewrite as `Equivalent to CHECK but accounts for potential false negatives`